crates/api_common/src/sensitive.rs

```diff
@@ -4,7 +4,7 @@ use std::{
   ops::{Deref, DerefMut},
 };
 
-#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Deserialize, Serialize)]
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Deserialize, Serialize, Default)]
```

Comment:
Necessary to make derive(Default) work for some of the structs, even though it obviously doesnt make so much sense. Hopefully the error messages are clear enough.